shoop/admin/templates/shoop/admin/contact_groups/edit.jinja

```diff
@@ -6,23 +6,15 @@
     {% call content_with_sidebar(content_id="contact_group_form") %}
         <form method="post" id="contact_group_form">
             {% csrf_token %}
-            {% call content_block(_("General information"), "fa-info-circle") %}
-                {{ language_dependent_content_tabs(form) }}
-                {{ render_monolingual_fields(form) }}
-            {% endcall %}
+            {% include form.form_defs["base"].template_name %}
```

Comment:
Use ``{% for form_def in form.form_defs.values() instead %}`` then you can use form_def.template_name in include.